languages/elixir/exercises/concept/charlists/.meta/design.md

```diff
@@ -0,0 +1,43 @@
+## Learning objectives
+
+After completing this exercise, the student should:
+
+- Know what charlists are and when they are used.
+- Know what chars are (code points).
+- Know how to use `?` to get a character's code point.
+- Know how to use `case` to match a given expression against the given clauses.
+
+## Out of scope
+
+- Changing charlists to strings and strings to charlists.
+- Pin operator.
+- Ranges.
+
+## Prerequisites
+
+This exercise's prerequisites are:
+
+- `lists`
+- `recursion`
+- `pattern-matching`
+- `guards`
+
+## Concepts
+
+The concepts this exercise unlocks are:
+
+- `charlists`
+- `case`
+
+## Representer
+
+This exercise does not require any specific representation logic to be added to the [representer][representer].
```

Comment:
For the C# and F# tracks, I've now opted to [leave out the representer or analyzer sections](https://github.com/exercism/v3/pull/1548) if they don't actually have any meaningful content. In this case, the representer section could be removed, but the analyzer does have useful contents so should stay. Whether you like to remove sections is up to you, but I just wanted to point it out :)